R/if_not_else_linter.R

```diff
@@ -91,10 +91,7 @@ if_not_else_linter <- function(exceptions = c("is.null", "is.na", "missing")) {
     if_lints <- xml_nodes_to_lints(
       if_expr,
       source_expression = source_expression,
-      lint_message = paste(
-        "In a simple if/else statement,",
-        "prefer `if (A) x else y` to the less-readable `if (!A) y else x`."
-      ),
+      lint_message = "Prefer `if (A) x else y` to the less-readable `if (!A) y else x` in a simple if/else statement.",
```

Comment:
Do we have an issue around that already? I thought so but didn't find it recently. If not, would you mind authoring the issue? I think you are the bigger advocate :)

Reply:
BTW, now I tend to use \` only when it removes ambiguity of mixing code with text, e.g. this is usually not needed for a single function call, but useful here since the code uses a lot of whitespace and has common English words `if`/`else`; the `is.numeric(.) || is.integer(.)` case is more borderline.

Agree that styling this with {cli} will allow us to be more consistent without considering tradeoffs like this.